### PR TITLE
Project Completion

### PR DESCRIPTION
**Pull request by Zibilyonik**
The project is completed with the required features.

### The app provides:
- Information on the SpaceX rockets, dragons and missions, retrieved from an external API
- Images retrieved from flickr_images on API
- Reservation button for each of the projects presented
- A header for users to switch between the tabs
- A neat table-style design to have a clean and presentable syle

### Users can:
- Click on reserve buttons to reserve their place in the upcoming rocket launches, missions and dragon launches
- Visit their profile to see the reservations they've made for the upcoming projects
- Cancel their reservations for the projects as they wish

### Main page view:
image

### Dragons screen view:
image

### Missions screen view:
image

### Profile screen view:
image

### Video Description:
[Google Drive Link](https://drive.google.com/file/d/1XBqgtK5_6PkkRqze9iCV1zerPhB9IeM2/view?usp=sharing)

### CODE REVIEW

**Comment by Metsanpeitto** [comment]
# Status: ✔️ Project Approved ✔️ 

Hello @Zibilyonik! 

**Your project is complete now, everything works correctly and all the features were implemented and totally functional.
You can proceed to merge. Good luck !**

go with the flow

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Zibilyonik** [comment]
Test

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (vichuge) [inline .babelrc:1]: - [ ] This is a test inline comment.

**Review by vichuge — COMMENTED** [review]
# This is a test to check if I can check a merged project.

**Pull request merged**